llvm/lib/Target/AMDGPU/SIInstructions.td

```diff
@@ -1097,7 +1097,7 @@ def : Pat <
 multiclass f16_fp_Pats<Instruction cvt_f16_f32_inst_e64, Instruction cvt_f32_f16_inst_e64> {
   // f16_to_fp patterns
   def : GCNPat <
-    (f32 (f16_to_fp i32:$src0)),
+    (f32 (any_f16_to_fp i32:$src0)),
```

Comment:
Should we replace more `f16_to_fp` to `any_f16_to_fp` in this file and other target files?

Reply:
Yes, with appropriate tests added alongside it